<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stock_Control</class>
 <widget class="QDialog" name="Stock_Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>838</width>
    <height>655</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>838</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stock Control</string>
  </property>
  <widget class="QLabel" name="lbl_Emoticon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources/images.qrc">:/emoticons/service.png</pixmap>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>703</x>
     <y>610</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Stock_Total_Value">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>620</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_Search_Part">
      <property name="text">
       <string>Search Part</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_Part_Name">
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Search entering the Part Name. You list all with: *</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>811</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="llb_Double_Click_Part">
      <property name="text">
       <string>Double click the part to change values</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tbl_Parts">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>811</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="lbl_Full_Part_Description">
      <property name="text">
       <string>Part Full Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txt_Part_Description">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tbl_Parts</tabstop>
  <tabstop>txt_Part_Description</tabstop>
 </tabstops>
 <resources>
  <include location="resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
